include/wally_psbt.h

```diff
@@ -303,6 +303,30 @@ WALLY_CORE_API int wally_psbt_input_keypath_add(
     const uint32_t *child_path,
     size_t child_path_len);
 
+/**
+ * Convert and add a pubkey/keypath to a taproot input.
+ *
+ * :param input: The input to add to.
+ * :param pub_key: The pubkey to add.
+ * :param pub_key_len: Length of ``pub_key`` in bytes. Must be `EC_XONLY_PUBLIC_KEY_LEN`.
+ * :param tapleaf_hashes: Series of 32-byte leaf hashes.
+ * :param tapleaf_hashes_len: Length of ``tapleaf_hashes`` in bytes.
+ * :param fingerprint: The master key fingerprint for the pubkey.
+ * :param fingerprint_len: Length of ``fingerprint`` in bytes. Must be `BIP32_KEY_FINGERPRINT_LEN`.
+ * :param child_path: The BIP32 derivation path for the pubkey.
+ * :param child_path_len: The number of items in ``child_path``.
+ */
+WALLY_CORE_API int wally_psbt_input_tap_keypath_add(
```

Comment:
Use `taproot` rather than `tap` to match `wally_tx_get_btc_taproot_signature_hash`.

Reply:
done